process-automation/latest/model/scripts.md

```diff
@@ -215,10 +215,6 @@ const formService = new FormService();
 const form = formService.getFormDefinition(formId);
 ```
 
-You can access a demo project found in the [script-tests](./doc/script-tests.zip) documentation. You can test the scripts capabilities by importing it into the `Modeling-app`.
-
-**Note:** The examples in the documentation that use `groupsApi` and `nodesApi` require the script client to be in the `ALFRESCO_ADMINISTRATORS` group to work.
```

Comment:
Sorry for the changes, but I don't think we have to remove the note because it's important. What do you think? I would remove the first part related to the demo access because it's something that we have in the git hub but not the note.

Reply:
No probs :-) 

Cheers, yeah it needs none or both because they relate to each other. 
I think leave the link in or we can change the link to the demo in github?
If so whats the link? 

Thanks!

Reply:
It's true. Ok, remove both. Sorry. Thanks!